### ISSUE TITLE
getting_started.ipynb last cell (sqlite load suggested change)

### ISSUE THREAD

**Issue by x2apps** [pymc-devs/pymc#943]
Suggested change for last executable cell in  https://github.com/pymc-devs/pymc3/blob/master/docs/notebooks/getting_started.ipynb

``` python
from pymc3.backends.sqlite import load

# with basic_model:
with Model() as model_glm_logistic:
    glm('y ~ x1', df_logistic, family=Binomial())
    trace_loaded = load('trace.sqlite')
```

The above will work with the test below, but the current code gives an 'x1' error:

``` python
summary(trace_loaded, vars=['x1'])
```

**Comment by fonnesbeck** [comment]
I can't replicate the error you are reporting. Can you ensure you are running the current master? Also, which version of Python are you using?

**Comment by x2apps** [comment]
Python 3.5.1
IPython 4.0.3
Used pip3 and re-installed pymc3: pymc3 3.0

Current code in notebook with error output

``` python
from pymc3.backends.sqlite import load

with basic_model:
    trace_loaded = load('trace.sqlite')
summary(trace_loaded, vars=['x1'])
```

KeyError                                  Traceback (most recent call last)
<ipython-input-31-b5f2d5c52c57> in <module>()
----> 1 summary(trace_loaded, vars=['x1'])

/usr/local/lib/python3.5/site-packages/pymc3/stats.py in summary(trace, vars, alpha, start, batches, roundto, to_file)
    462     for var in vars:
    463         # Extract sampled values
--> 464         sample = trace.get_values(var, burn=start, combine=True)
    465 
    466         fh.write('\n%s:\n\n' % var)

/usr/local/lib/python3.5/site-packages/pymc3/backends/base.py in get_values(self, varname, burn, thin, combine, chains, squeeze)
    242         try:
    243             results = [self._straces[chain].get_values(varname, burn, thin)
--> 244                        for chain in chains]
    245         except TypeError:  # Single chain passed.
    246             results = [self._straces[chains].get_values(varname, burn, thin)]

/usr/local/lib/python3.5/site-packages/pymc3/backends/base.py in <listcomp>(.0)
    242         try:
    243             results = [self._straces[chain].get_values(varname, burn, thin)
--> 244                        for chain in chains]
    245         except TypeError:  # Single chain passed.
    246             results = [self._straces[chains].get_values(varname, burn, thin)]

/usr/local/lib/python3.5/site-packages/pymc3/backends/sqlite.py in get_values(self, varname, burn, thin)
    222 
    223         self.db.connect()
--> 224         shape = (-1,) + self.var_shapes[varname]
    225         statement = TEMPLATES[action].format(table=varname)
    226         self.db.cursor.execute(statement, statement_args)

KeyError: 'x1'

Here I changed the notebook's code:

``` python
from pymc3.backends.sqlite import load

# with basic_model:
with Model() as model_glm_logistic:
    glm('y ~ x1', df_logistic, family=Binomial())
    trace_loaded = load('trace.sqlite')
summary(trace_loaded, vars=['x1'])
```

x1:

  Mean             SD               MC Error         95% HPD interval

---

  0.000            0.129            0.002            [-0.248, 0.251]

  Posterior quantiles:
  2.5            25             50             75             97.5
  |--------------|==============|==============|--------------|

  -0.252         -0.084         0.001          0.087          0.248

**Comment by ds7788** [comment]
@x2apps I get this error as well. Did yo figure out what was wrong with it? I also get it with just doing a simple model.

**Comment by usptact** [comment]
Looks like this has been addressed in #2894 

I don't see the last two lines in the current version.

**Comment by junpenglao** [comment]
Thanks @usptact, closing this.
